modules/ROOT/pages/monitoring-dashboards.adoc

```diff
@@ -16,139 +17,142 @@ The app monitoring dashboard helps you answer questions about your integrated ap
 
 === Server Metrics
 
-The graphs in this server's dashboard cover different metrics related to memory and CPU usage. You can see these metrics at a high level on the *Overview* tab, or you can view the *Breakdown* to see a detailed series of metrics that tracks each part of the memory separately.
+The graphs in this server dashboard provide metrics related to memory and CPU usage. You can see these metrics at a high level on the *Overview* tab or you can click *Breakdown* to see detailed memory usage metrics.
 
-The graphs in this dashboard display memory usage levels in both the *Overview* and the *Breakdown* tabs, and track these values:
+The graphs in this dashboard track these values:
 
-* *Used* - The amount of memory (in MB) that is currently in use. 
-If the Java VM attempts to increase the used memory to be greater than committed memory, even if the amount of used memory is increased to below the total memory, memory allocation fails.
-* *Committed* - The amount of memory (in MB) that's available for use by the Java VM. This amount varies over time. The amount of committed memory is always greater than or equal to the amount of used memory.
-* *Total* - The maximum amount of memory (in MB) for use by the server. The maximum amount can change or be undefined. 
+* *Usage*
++
+The amount of memory (in MB) currently in use. 
++
+If the Java VM attempts to increase the used memory to be greater than committed memory,  memory allocation fails even if the amount of used memory is increased to below the total memory.
+* *Total*
++
+The maximum amount of memory (in MB) for use by the server. The maximum amount can change or be undefined. 
+* *Committed*
++
+The amount of memory (in MB) available for use by the Java VM. This amount varies over time. The amount of committed memory is always greater than or equal to the amount of used memory.
 
 === JVM Heap Memory
 
-The heap is the area where all Java objects reside. The heap is created when the JVM starts, and can scale up or down in size without interrupting the applications in it. When the heap becomes full, <<garbage_collection,garbage is collected>> to clear objects that are no longer used and make space for new objects.
-
-The non-heap memory stores per-class structures such as a runtime constant pool, field and method data, and the code for methods and constructors.
+The heap is the area where all Java objects reside. The heap is created when the JVM starts and can scale up or down in size without interrupting the applications in it. When the heap becomes full, <<garbage_collection,garbage is collected>> to clear objects that are no longer used and make space for new objects.
 
-This metric in the *Overview* is the total heap memory usage in MB. See the <<Breakdown Tab,*Breakdown* tab>> for details about each of its components.
+The non-heap memory stores per-class structures such as a runtime constant pool, field, and method data, and the code for methods and constructors.
```

Comment:
Is "field" supposed to be "field data"? Wondering about the parallelism among pool, field, and data....